workflow-service/src/test/java/com/redhat/parodos/workflow/execution/scheduler/WorkFlowSchedulerServiceImplTest.java

```diff
@@ -0,0 +1,94 @@
+package com.redhat.parodos.workflow.execution.scheduler;
+
+import com.redhat.parodos.workflows.work.Work;
+import com.redhat.parodos.workflows.work.WorkContext;
+import com.redhat.parodos.workflows.workflow.SequentialFlow;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mockito;
+import org.springframework.scheduling.TaskScheduler;
+import org.springframework.scheduling.Trigger;
+import org.springframework.scheduling.support.CronTrigger;
+
+import javax.validation.constraints.AssertFalse;
+import java.time.Instant;
+import java.util.concurrent.ScheduledFuture;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class WorkFlowSchedulerServiceImplTest {
+
+	private TaskScheduler taskScheduler;
+
+	private WorkFlowSchedulerServiceImpl service;
+
+	private Work work;
+
+	private SequentialFlow workFlow;
+
+	@BeforeEach
+	public void each_init() {
```

Comment:
pls follow the method naming convention - https://google.github.io/styleguide/javaguide.html#s5.2.3-method-names ;-)